src/html/jsdoc.rs

```diff
@@ -154,9 +154,28 @@ pub fn markdown_to_html(
     let mut ammonia_builder = ammonia::Builder::default();
 
     ammonia_builder
-      .add_tags(["video"])
+      .add_tags(["video", "button", "svg", "path"])
       .add_generic_attributes(["id"])
+      .add_tag_attributes("button", ["onclick"])
```

Comment:
Very bad :sweat_smile: 

Maybe add a `data-copy="thing to copy"` attribute to the button, and then register a global click handler on the document and when invoked, check whether the target has a `data-copy`, and if it does, `e.preventDefault()` and copy?

We can use that for all the copy buttons then.